components/automate-ui/src/app/page-components/profile/profile.component.html

```diff
@@ -13,10 +13,15 @@
 <chef-click-outside omit="dropdown-toggle" (clickOutside)="closeDropdown()">
   <chef-dropdown class="dropdown" [attr.visible]="dropdownVisible" (keydown.esc)="closeDropdown()">
     <ul class="dropdown-list">
-      <li class="dropdown-list-item">
-        Signed in as <strong class="dropdown-display-name">{{ displayName }}</strong>
-      </li>
+      <app-authorized not [allOf]="['/auth/users/{username}', 'get', userName]">
+        <li class="dropdown-list-item">
+          Signed in as <strong class="dropdown-display-name">{{ userName }}</strong>
+        </li>
+      </app-authorized>
```

Comment:
Non-local user doe not have access to '/auth/users/'. For these users, we use the userName as the display name and do not show the profile page.